### PR TITLE
feat: added extraEnvRaw variable to load values from other secrets in Helm chart

### PR DESCRIPTION
**Pull request by diffnull**
### SUMMARY

Added ability to load additional environment variables from third-party secrets, e.g. load DB password from zalando operator secret.

Solve #16245

### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [ ] Changes UI
- [ ] Includes DB Migration (follow approval process in [SIP-59](https://github.com/apache/superset/issues/13351))
  - [ ] Migration is atomic, supports rollback & is backwards-compatible
  - [ ] Confirm DB migration upgrade and downgrade tested
  - [ ] Runtime estimates and downtime expectations provided
- [x] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

File: helm/superset/values.yaml [hunk 1]
```diff
@@ -54,11 +54,21 @@ extraEnv: {}
...
 
-   # OAUTH_HOME_DOMAIN: ..
+  # OAUTH_HOME_DOMAIN: ..
   # # If a whitelist is not set, any address that can use your OAuth2 endpoint will be able to login.
   # #   this includes any random Gmail address if your OAuth2 Web App is set to External.
   # OAUTH_WHITELIST_REGEX: ...
 
+## Extra environment variables in RAW format that will be passed into pods
+##
+extraEnvRaw: {}
```
Comment (craig-rueda) [inline helm/superset/values.yaml:64]: I think the default should be set to empty here. Using `{}` as the default is a little misleading as it's a dictionary type whereas we need a list instead.

Comment (elyzov) [reply helm/superset/values.yaml]: fixed

**Review by craig-rueda — APPROVED** [review]

**Comment by elyzov** [comment]
@eschutho tell me please, what should I do next to merge the PR?

**Comment by craig-rueda** [comment]
Looks like you need a version bump

**Comment by elyzov** [comment]
@craig-rueda Ok, thanks, I've just bumped a version

**Comment by elyzov** [comment]
@craig-rueda seems like now it's all right, can we merge the PR?

**Pull request merged**